example.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file? Do things work without it? We should not need config files like this

Reply:
This seems to be a file created by IntelliJ to keep the module configuration. The program should work without it.

Reply:
Lets please remove this file.